src/tmx/FindTmxMySQL.cmake

```diff
@@ -86,6 +86,6 @@ SET (MYSQLCPPCONN_INCLUDE_DIRS ${MYSQLCPPCONN_INCLUDE_DIR} )
 
 MESSAGE (STATUS "Found MySQL C++ Connector: ${MYSQLCPPCONN_LIBRARY}")
 INCLUDE (FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS (MYSQLCPPCONN DEFAULT_MSG MYSQLCPPCONN_INCLUDE_DIR MYSQLCPPCONN_LIBRARY)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS (TmxMySQL DEFAULT_MSG MYSQLCPPCONN_INCLUDE_DIR MYSQLCPPCONN_LIBRARY)
```

Comment:
What is the purpose and impact of this change.

Reply:
This was changed as there was a warning being shown with it the old way.  I can show and we can document here.

Reply:
FYI i pasted the error seen for this if left as is.

```CMake Warning (dev) at /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args`
  (MYSQLCPPCONN) does not match the name of the calling package (TmxMySQL).
  This can lead to problems in calling code that expects `find_package`
  result variables (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  FindTmxMySQL.cmake:89 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  CMakeLists.txt:51 (FIND_PACKAGE)
This warning is for project developers.  Use -Wno-dev to suppress it.```